controllers/default.py

```diff
@@ -71,9 +76,11 @@ def api():
     return Collection(db).process(request,response,rules)
 
 def register():
+    auth.settings.register_next = 'confirm.html'
     response.title = 'iDibo Registration'
-    return dict()
+    return dict(form = auth.register())
 
+@auth.requires_login()
```

Comment:
is the api() function above still needed? you have the same annotation on both, I think we could refactor this